run_offline.md

```diff
@@ -0,0 +1,20 @@
+### How to run the Installation offline without macOS
+
+- Download the ventura file (InstallAssistant.pkg) from https://mrmacintosh.com/macos-ventura-13-full-installer-database-download-directly-from-apple/(https://mrmacintosh.com/macos-ventura-13-full-installer-database-download-directly-from-apple/)
+- Create an iso file `InstallAssistant.iso` with the downloaded InstallAssistant.pkg and `run_offline.sh` in the `scripts/run_offline.sh` files
```

Comment:
How are you creating the `InstallAssistant.iso` file?

Reply:
using this cmd 
`genisoimage -allow-limited-size -l -J -r -iso-level 3 -V InstallAssistant -o InstallAssistant.iso path/to/InstallAssistant.pkg path/to/OSX-KVM/scripts/run_offline.sh`